documentation/numerical/equation.rst

```diff
@@ -21,7 +21,7 @@ Examples of such systems are wide ranging, but include problems that
 exhibit a combination of diffusing and reacting species, as well as
 such diverse problems as determination of the electric potential in
 heart tissue, of fluid flow, stress evolution, and even the
-Schroedinger equation.
```

Comment:
If you're going to be this pedantic, why misspell the reference label?

Reply:
True.